test/component_test.go

```
@@ -0,0 +1,137 @@
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/stretchr/testify/assert"
+)
+
+type ComponentSuite struct {
+	helper.TestSuite
+}
+
+func (s *ComponentSuite) TestBasic() {
+	const component = "alb/basic"
+	const stack = "default-test"
+	const awsRegion = "us-east-2"
```

golangci-lint (1.62.2)

const `awsRegion` is unused

(unused)